docs/staking/intro.rst

```diff
@@ -0,0 +1,312 @@
+Using Gaia
+==========
+
+This project is a demonstration of the Cosmos Hub with staking functionality; it is
+designed to get validator acquianted with staking concepts and procedure.
+
+Potential validators will be declaring their candidacy, after which users can
+delegate and, if they so wish, unbond. This can be practiced using a local or
+public testnet.
+
+Setup Testnet
+-------------
+
+The first thing you'll want to do is either `create a local testnet <./local-testnet.html>`__ or
+join a `public testnet <./public-testnet.html>`__. Either step is required before proceeding
+(although you should browse below and make some keys first).
```

Comment:
This whole paragraph is really confusing, the first thing you want to do is generate keys... I would change this paragraph to something along the lines: 

Before proceeding with the tutorials you will need to:
 - set up some [keys](link to the generating keys sections )
 - Either 
   -`create a local testnet <./local-testnet.html>`
   - or, join a `public testnet <./public-testnet.html>`
Afterwords you may proceed with testing out the functionality.